src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -0,0 +1,194 @@
+                $integrationAccountName = $config.Arcus.IntegrationAccount.Name
+				$schemasFolder = "$PSScriptRoot\Files\IntegrationAccount\Schemas"
+                $executionDateTime = Get-Date
+
+                try {
+                    # Act
+                    Set-AzIntegrationAccountSchemas -ResourceGroupName $resourceGroupName -Name $integrationAccountName -SchemasFolder $schemasFolder
+
+                    # Assert
+                    foreach ($schema in Get-ChildItem("$schemasFolder") -File) {
+                        $expectedSchemaName = $schema.Name
+                        
+                        $actual = Get-AzIntegrationAccountSchema -ResourceGroupName $resourceGroupName -Name $integrationAccountName -SchemaName $expectedSchemaName
+                        $actual | Should -Not -BeNullOrEmpty
+                        $actual.CreatedTime.ToString("yyyy-MM-ddTHH:mm:ss") | Should -Be $actual.ChangedTime.ToString("yyyy-MM-ddTHH:mm:ss")
+                        $actual.CreatedTime | Should -BeGreaterOrEqual $executionDateTime
+                    }
+
+                } finally {
+                    foreach ($schema in Get-ChildItem("$schemasFolder") -File) {
+                        $expectedSchemaName = $schema.Name
+                        
+                        Remove-AzIntegrationAccountSchema -ResourceGroupName $resourceGroupName -Name $integrationAccountName -SchemaName $expectedSchemaName -Force
+                    }
+                }
+            }
+            It "Create multiple schemas, without extension, located in a folder in an Integration Account" {
+                # Arrange
+                $resourceGroupName = $config.Arcus.ResourceGroupName
+                $integrationAccountName = $config.Arcus.IntegrationAccount.Name
+				$schemasFolder = "$PSScriptRoot\Files\IntegrationAccount\Schemas"
+                $executionDateTime = Get-Date
+
+                try {
+                    # Act
+                    Set-AzIntegrationAccountSchemas -ResourceGroupName $resourceGroupName -Name $integrationAccountName -SchemasFolder $schemasFolder -RemoveFileExtensions
+
+                    # Assert
+                    foreach ($schema in Get-ChildItem("$schemasFolder") -File) {
+                        $expectedSchemaName = $schema.BaseName
+                        
+                        $actual = Get-AzIntegrationAccountSchema -ResourceGroupName $resourceGroupName -Name $integrationAccountName -SchemaName $expectedSchemaName
+                        $actual | Should -Not -BeNullOrEmpty
+                        $actual.CreatedTime.ToString("yyyy-MM-ddTHH:mm:ss") | Should -Be $actual.ChangedTime.ToString("yyyy-MM-ddTHH:mm:ss")
+                        $actual.CreatedTime | Should -BeGreaterOrEqual $executionDateTime
+                    }
+
+                } finally {
+                    foreach ($schema in Get-ChildItem("$schemasFolder") -File) {
```

Comment:
To bad Pester doesn't have `Assert.All` or `Assert.Collection` stuff 😄 .

Reply:
Yeah 💯